### PR TITLE
 data.md, driver.md, evm.md, verification.md: unify java and ocaml semantics some

### PR DESCRIPTION
**Pull request by dwightguth**
Do not review yet

### CODE REVIEW

File: data.md [hunk 1]
```diff
@@ -205,21 +169,8 @@ Care is needed for `^Word` to avoid big exponentiation.
...
-```
-
-RV-K has a more efficient power-modulus operator.
-
-```{.k .ocaml .node}
     rule W0 ^Word W1 => powmod(W0, W1, pow256)
-```
 
-```k
     rule powmod(W0, W1, W2) => W0 ^%Int W1 W2 requires W2 =/=Int 0
```
Comment (ehildenb) [inline data.md]: Should we just go straight through `_^%Int__` instead of through `powmod` now? It seems unneeded.

**Comment by dwightguth** [comment]
@ehildenb I can't just remove the powmod function as it is used by the MODEXP precompiled contract, and the behavior of returning zero when the modulus is zero is necessary there. Do you want me to change `^Word` though to refer directly to ^%Int?

**Comment by ehildenb** [comment]
@dwightguth yeah I think it makes sense to switch the uses of `powmod` to `_^%Int__` because essentially `powmod` has become just another name for `_^%Int__`. It will just add an (admittedly small) reduction step to go through `powmod`.

**Comment by dwightguth** [comment]
Fixed. @ehildenb can you re-review?

**Comment by dwightguth** [comment]
Jenkins: test this please

**Pull request merged**